src/coreclr/tools/Common/Compiler/DevirtualizationManager.cs

```diff
@@ -212,5 +212,7 @@ protected virtual MethodDesc ResolveVirtualMethod(MethodDesc declMethod, DefType
         /// </remarks>
         public virtual bool CanConstructType(TypeDesc type) => true;
 #endif
+
+        public virtual TypeDesc[] GetImplementingClasses(TypeDesc type) => null;
```

Comment:
We should put this two lines up into the `!READYTORUN` block - for now this can never return a meaningful answer for R2R and we shouldn't have it available.

We could potentially implement it in R2R if we're building a trimmed composite assembly and trimming removed all possibilities to load new types at runtime (no Assembly.LoadFrom, Ref.Emit, etc.), and we have a whole program view. It's a ton more work but doable in theory if we find this having significant improvement for NAOT.